example/helloWorld/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+project(alpaka-helloworld)
```

Comment:
The license header is missing. Just copy it from the other examples and replace the name.